src/azure-cli/HISTORY.rst

```diff
@@ -30,6 +30,12 @@ Release History
 * Fix `vm create` failure in Azure Stack profile.
 * vm monitor metrics tail/list-definitions: support query metric and list definitions for a vm.
 
+**Cdn**
```

Comment:
your history is very stale. pls run `git rebase dev` for your branch to update

Reply:
I have created new PR to fix the versioning issue.